rfcs/rfc-proposal-0001.md

```diff
@@ -0,0 +1,155 @@
+
+    Once the proposal template has been completed, a PR should be raised with containing the proposal document along with any supporting documentation that the proposal needs. Supporting documentation can take the form of code samples, draft documentation, example usages or proposed timelines according to the content of the proposal.
+
+    If a `WIP` proposal is already raised for the proposal, the `WIP` should be removed from the title and the `Work in Progress` label should be removed from the PR. 
+
+    The PR should be labeled with the `RFC` label. Proposers should ensure they have assigned the PR to themselves if they are a member of the repo team. If they are not a team member, the PR comments should include the proposers details, including the team they are on (if Mozilla) or whether they are a partner or a contributor, and how they are consuming GeckoView.
+
+    The PR Title should be the same as the title included in the proposal document. 
+
+    Critical reviewers should be named in the description of the PR alongside a checkbox. Critical reviewers are reviewers without whose agreement the proposal cannot be accepted. If new critical reviewers are discovered during the review process, their names should be added at that time.
+
+    Once the PR has been submitted, a link to the proposal PR should be added in a comment to the Bugzilla or Github issue
+
+4. Requesting feedback on the proposal
+    After submitting the PR containing the proposal and any supporting documents, the GeckoView team should be added to the list of reviewers. All proposals must include the GeckoView team as reviewers. Proposals that have not been reviewed by the GeckoView team will not be accepted.
+
+    If there are other interested parties who also need to review the proposal, they should also be included in the reviewers. If the interested party is not on GitHub then the following methods should be used to request feedback:
+
+    - Email reviewer directly and request feedback giving link to PR.
+    - Add link to PR to the Bugzilla issue and NI that reviewer from there.
+    - Contact the reviewer through Slack or IRC and request that they review the proposal.
+
+    Advice should be provided in the `README.md` inside `rfcs` to instruct proposers to make clear the importance of the feedback for the issue when contacting a reviewer directly. If the reviewer has been identified as a Critical Reviewer then any communication should include the fact that the reviewers feedback is required before the proposal can be accepted.
+
+5. Responding to feedback on the proposal
+
+    On receipt of feedback on a proposal, the proposal should be updated to take the feedback into account. This can be done either as a new commit or amending the original commit to include the changes. The reviewer should be notified that their feedback has been addressed. Responding to their comment on the PR should be sufficient.
+
+6. Entering the final review phase
```

Comment:
This feels a little heavy to start. Maybe we could start with having similar rules to the ones that we have for API changes (2 reviewers from GV team) + 1 representatives of any team that might be involved (fenix, a-c, addons, etc)?